### DISCUSSION TITLE
REQ: Implement TauntyArmordillo/interaccel mouse acceleration curve

### DISCUSSION THREAD

**Discussion by meag** [ezQuake/ezquake-source#504] [Ideas]
Please ensure that your request is predominantly client-based, and not better suited to being logged with the server or mod aspects of QuakeWorld (e.g. mvdsv or ktx)

**Describe the solution you'd like**
![image](https://user-images.githubusercontent.com/10936291/89755306-dfc04e00-dad6-11ea-9fd5-fb4b939661df.png)

https://github.com/KovaaK/InterAccel/pull/18/files#diff-3d8a6a8ef770d674b66ade1ef5ace04dR695-R718

**Comment by LuzHansen** [comment]
I’d like to see the TauntyArmordillo interaccel mouse acceleration curve implemented as an optional client-side setting. The goal would be to replicate its specific acceleration behavior as closely as possible, including how it scales with low-speed precision and ramps up at higher movement speeds. Ideally this would be configurable through existing mouse cvars, or a dedicated toggle with exposed parameters for sensitivity, accel strength, and offset. King Billy https://kingbillyonlinecasino.com/ brings simple play and exciting wins. Keeping it fully client-based ensures consistency without affecting server-side behavior, while giving players a modern, customizable input feel.

**Comment by JesseReed123** [comment]
Por favor, asegúrate de que tu solicitud se enfoque principalmente en el cliente y no sea más adecuada para registrarse en el servidor o en los aspectos del mod de QuakeWorld, como mvdsv o ktx. La idea es que cualquier funcionalidad o https://roobetcasino.online/ mejora solicitada se pueda implementar y usar directamente desde el cliente, sin depender de cambios en el servidor. Esto ayudará a que la propuesta sea más práctica, accesible para todos los usuarios y más fácil de integrar en futuras versiones del cliente.
